wireguard-broker/src/api/server.rs

```diff
@@ -95,3 +126,60 @@ where
         Ok(())
     }
 }
+
+// We can only include this test if this feature is enabled, because otherwise
+// brokers::netlink::SetPskError is not defined.
+#[cfg(all(feature = "experiment_api", target_os = "linux"))]
```

Comment:
Since we mock the WireGuard broker anyway, shouldn't this be able to run on non-linux, too?

Reply:
Indeed, will fix this in the next commit/amendment to the last commit.